### ISSUE TITLE
FNX3-16755 ⁃ Show A-C version on about page

### ISSUE THREAD

**Issue by nojunpark** [mozilla-mobile/fenix#3055]
### Why/User Benefit/User Problem
If the A-C version number is readily available, then it would make things easier to investigate a-c issues on Fenix

### What/Requirements
Have A-C version shown under 'About Firefox Preview' page

### Acceptance Criteria (how do I know when I’m done?)
Have A-C version shown under 'About Firefox Preview' page

**Comment by jonalmeida** [comment]
For the person working on this, these attributes are already exposed from AC:
```
println(mozilla.components.Build.applicationServicesVersion)
println(mozilla.components.Build.gitHash)
println(mozilla.components.Build.version)
```

**Comment by cadeyrn** [comment]
Dupe of #2927?

**Comment by jonalmeida** [comment]
> Duplicate of #2927?

Nice catch Sorin! I'm going to close that one off as a dupe of this since it's more visible.

**Comment by sarentz** [comment]
Not sure why this is a blocker. Nice to have, not blocking.

**Comment by jonalmeida** [comment]
Didn't noticed the labels earlier; this is also not blocked by anything.

**Comment by vesta0** [comment]
This is definitely not a blocker.

**Comment by softvision-miralobontiu** [comment]
Verified that on the latest build 1.0.1926 (#11760617), in "About Firefox Preview" the AC: 1.0.1 is displayed.
Tested with Samsung Galaxy S8 (Android 9), and Nexus 5 (Android 6.0.1).
